packages/stems/src/components/Popup/types.ts

```diff
@@ -35,7 +35,7 @@ export type PopupProps = {
   /**
    * Children to render inside the Popup
    */
-  children: ReactChild
+  children: ReactNode
```

Comment:
`ReactChild` is deprecated

Reply:
should this be ReactElement actually? i think this ensures its a single child and is reffable?

Reply:
I think `ReactNode` is the default for ComponentPropsWithChildren. See also https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement

Reply:
word, yeah i guess we receive the containerRef as a prop, so its actual children can just be whatever